CHANGELOG.md

```diff
@@ -7,13 +7,33 @@ OF 0.9.0
           / modified
 
 ------------------------------------------------------------------------------
+GENERAL
+-------
+
++ added a new Project Generator
++ added CODE_OF_CONDUCT.md based on the Contributor Covenant and Berlin Code of Conduct, iteratively refined through group discussion
+/ code cleanup (fixed many typos, warnings, memleaks, implicit conversions, types, const-correctness, duplicated and missing definitions/functions, removed internal use of deprecated methods and unnecessary code) and move towards C++11
```

Comment:
i think it's better to have "move towards c++11" as a different line. seems important enough to be a thing alone rather than part of code cleanup